tests/monitor/test_dbt_runner.py

```diff
@@ -46,3 +47,21 @@ def test_dbt_runner_run(mock_subprocess_run, model, full_refresh):
     if full_refresh:
         assert '--full-refresh' in mock_subprocess_run.call_args[0][0]
 
+@pytest.mark.parametrize("dbt_vars", [
+    (None),
+    ({"key1": "x", "key2": "y", "key3": "z"}),
+])
+@mock.patch('subprocess.run')
+def test_dbt_runner_run(mock_subprocess_run, dbt_vars):
```

Comment:
You can combine this test with the previous one 🙂 
Because `dbt_vars` is not affected by the other params, you can write a single test like:
```python
@pytest.mark.parametrize("model,full_refresh,dbt_vars", [
    ('m1', True, None),
    ('m1', False, {"key1": "x", "key2": "y", "key3": "z"}),
    (None, True, {"key1": "x", "key2": "y", "key3": "z"}),
    (None, False, None),
])
@mock.patch('subprocess.run')
def test_dbt_runner_run(mock_subprocess_run, model, full_refresh, dbt_vars):
    project_dir = 'proj_dir'
    profiles_dir = 'prof_dir'
    expanded_dbt_vars = json.dumps(dbt_vars)
    dbt_runner = DbtRunner(project_dir=project_dir, profiles_dir=profiles_dir)
    dbt_runner.run(model, full_refresh=full_refresh, vars=dbt_vars)
    mock_subprocess_run.assert_called()
    if model is not None:
        assert model in mock_subprocess_run.call_args[0][0]
    if full_refresh:
        assert '--full-refresh' in mock_subprocess_run.call_args[0][0]
    if dbt_vars is None:
        assert '--vars' not in mock_subprocess_run.call_args[0][0]
        assert expanded_dbt_vars not in mock_subprocess_run.call_args[0][0]
    if dbt_vars is not None:
        assert '--vars' in mock_subprocess_run.call_args[0][0]
        assert expanded_dbt_vars in mock_subprocess_run.call_args[0][0]
```

Reply:
Took the code